ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -369,10 +367,11 @@ def yield_data_models(self, dbt_files: DbtFiles) -> Iterable[DataModelLink]:
                         ),
                         datamodel=DataModel(
                             modelType=ModelType.DBT,
-                            description=manifest_node.get("description")
-                            if manifest_node.get("description")
+                            description=manifest_node.get("description"),
+                            path=f"{manifest_node['root_path']}/{manifest_node['original_file_path']}"
```

Comment:
IMO if we take this out here and put as if else block , would be easy to read

Reply:
Sure, will do that